backend/src/tasks/censysCertificates.ts

```diff
@@ -117,16 +110,28 @@ const downloadPath = async (
 };
 
 export const handler = async (commandOptions: CommandOptions) => {
-  const { chunkNumber, numChunks, organizationId } = commandOptions;
+  const { chunkNumber, organizationId } = commandOptions;
+
+  // Sanitizes numChunks to protect against arbitrarily large numbers
+  const numChucksRawValue = commandOptions.numChunks;
```

Comment:
I think there's a typo -- this should be "numChunksRawValue" not "numChucksRawValue"